Implement decimals in webUI

andrevmatos opened the pull request:
Use jQuery's Deferred to gather token webUi parallely, fetch token decimals, and use it across webUI to show and prompt numbers/balances with correct number of digits. Fix #87

pcppcp commented on microraiden/microraiden/webui/js/main.js:
```diff
@@ -121,7 +121,11 @@ function pageReady(json) {
       $('.tkn-name').text(token.name);
       $('.tkn-symbol').text(token.symbol);
-      $('.tkn-balance').attr("value", `${token.balance || 0} ${token.symbol}`);
+      const bal = parseFloat(token.balance || 0).toFixed(token.decimals || 0);
```
this should be 
```js
const bal = parseFloat(token.balance || 0) / Math.pow(10, token.decimals || 0);
```

andrevmatos commented:
It also fixes #109 , as it didn't show the error dialog only because of a small typo in a code that were also changed by this PR, so I fixed it here too.
